README.md

```diff
@@ -24,7 +24,10 @@ This open source project is fully backed by the Microsoft Healthcare team, but w
 - A FHIR mapping template uploaded to the template storage container. See [Configuration](./docs/Configuration.md) for more information.
 
 # Getting Started
-To get started you can deploy the [IoMT FHIR Connector for Azure](./docs/ARMInstallation.md) by itself or start with a complete [sandbox environment](./docs/Sandbox.md) that includes an instance of [IoT Central](https://azure.microsoft.com/en-us/services/iot-central/) with simulated devices and a deployed instance of the [Azure API for FHIR](https://docs.microsoft.com/en-us/azure/healthcare-apis/).
+To get started, there are a few options:
+1. Deploy the [IoMT FHIR Connector for Azure](./docs/ARMInstallationManagedIdentity.md) by itself for use with Azure API for FHIR and Azure Active directory in the same subscription

```

Comment:
Does it mean managed identity flavor of IoMT connector doesn't work if Azure API for FHIR and IoMT Connector are on different subscriptions? If yes, we may want to cover this use case in #2 line item.

Reply:
I believe so, but @namalu or @dustinburson can confirm

Reply:
I think it is more correct to say it won't work if they are on different tenants but in principle yes.